relay/api/resources.py

```diff
@@ -118,6 +118,8 @@ def get(self, network_address, a_address, b_address):
         graph = self.trustlines.currency_network_graphs[network_address]
         data = TrustlineSchema().dump(graph.get_account_sum(a_address, b_address)).data
         data.update({
+            'user': a_address,
+            'counterParty': b_address,
             'address': b_address,
```

Comment:
address is still being set here. not sure if that's on purpose or if it could be removed.

Reply:
can be removed in the future, I only left it so that this is not a breaking change.